.github/workflows/staticmajor.yml

```diff
@@ -1,23 +0,0 @@
-# Staticmajor: Static analyzer to catch leaking resources & other bad code patterns
```

Comment:
This file is also removed from `cosmos-sdk` `main` due to false postives: https://github.com/cosmos/cosmos-sdk/commit/9e9cc5299b2f636c43d0e9370a68861678080905